docs/extensions/guides/renderer-extension.md

```diff
@@ -374,20 +385,21 @@ The `feature` field must specify an instance which extends the abstract class `C
-The implementation of this method should install kubernetes resources using the `applyResources()` method, or by directly accessing the kubernetes api ([`K8sApi`](tbd)).
+The four methods listed above are defined as follows:
+
+* The `install()` method installs Kubernetes resources using the `applyResources()` method, or by directly accessing the [Kubernetes API](../api/README.md). Click **Install** on the cluster **Settings** page to call this method.
+
+* The `upgrade()` method upgrades the Kubernetes resources already installed, if they are relevant to the feature. Click **Upgrade** on the cluster **Settings** page to call this method.
+
+* The `uninstall()` method uninstalls Kubernetes resources using the [Kubernetes API](../api/README.md). Click **Uninstall** on the cluster **Settings** page to call this method.
 
-The `upgrade()` method is typically called by Lens when a user has indicated that this feature is to be upgraded (i.e. clicked **Upgrade** for the feature on the cluster settings page).
-The implementation of this method should upgrade the kubernetes resources already installed, if relevant to the feature.
+* The `updateStatus()` method provides the current status information in the `status` field of the `ClusterFeature.Feature` parent class. Lens periodically calls this method to determine details about the feature's current status. Consider using the following properties with `updateStatus()`:
 
-The `uninstall()` method is typically called by Lens when a user has indicated that this feature is to be uninstalled (i.e. clicked **Uninstall** for the feature on the cluster settings page).
-The implementation of this method should uninstall kubernetes resources using the kubernetes api (`K8sApi`)
+    * `status.currentVersion` and `status.latestVersion` may be displayed by Lens in the feature's description.
 
-The `updateStatus()` method is called periodically by Lens to determine details about the feature's current status.
-The implementation of this method should provide the current status information in the `status` field of the `ClusterFeature.Feature` parent class.
-The `status.currentVersion` and `status.latestVersion` fields may be displayed by Lens in describing the feature.
-The `status.installed` field should be set to true if the feature is currently installed, otherwise false.
-Also, Lens relies on the `status.canUpgrade` field to determine if the feature can be upgraded (i.e a new version could be available) so the implementation should set the `status.canUpgrade` field according to specific rules for the feature, if relevant.
+    * `status.installed` should be set to `true` if the feature is installed, and `false` otherwise.
+
+    * `status.canUpgrade` is set according to a rule meant to determine whether the feature can be upgraded. 
```

Comment:
"...which can involve `status.currentVersion` and `status.latestVersion` if desired"  (or better wording)